charts/gsp-cluster/values.yaml

```diff
@@ -134,18 +134,638 @@ fluentd-cloudwatch:
     requests:
       memory: 512Mi
   image:
-    tag: v1.3.2-debian-cloudwatch  # More recent image needed for fluentd-kubernetes-daemonset to avoid utf-8 encoding errors
+    tag: v1.7.4-debian-cloudwatch-1.0
   rbac:
     create: true
   awsRegion: eu-west-2
-  extraVars:
-    - "{ name: FLUENT_UID, value: '0' }"  # run fluentd as root as instructed by https://github.com/helm/charts/tree/master/incubator/fluentd-cloudwatch
   updateStrategy:
     type: RollingUpdate
+  volumeMounts:
+    - name: runlogjournal
+      mountPath: /run/log/journal
+      readOnly: true
+    - name: dmesg
+      mountPath: /var/log/dmesg
+      readOnly: true
+  volumes:
+    - name: runlogjournal
+      hostPath:
+        path: /run/log/journal
+    - name: dmesg
+      hostPath:
+        path: /var/log/dmesg
   tolerations:
     - key: node-role.kubernetes.io/master
       effect: NoSchedule
   priorityClassName: gsp-critical
+  data:
+    fluent.conf: |
+      @include containers.conf
+      @include systemd.conf
+      @include host.conf
+
+      <match fluent.**>
+        @type null
+      </match>
+    containers.conf: |
```

Comment:
shame this all has to go in the values file, as it's quite hard to read ... but not sure there is much that can be done about that.

Reply:
These changes are needed to override the default fluentd config file which doesn't ship the kubelet log messages from journald to AWS CloudWatch.